### PR TITLE
Add pytests to CI

### PR DESCRIPTION
**Pull request by lcoombe**
* Adding some pytests to the CI to run the LongStitch pipeline and compare to the expected outputs

### CODE REVIEW

**Review by theottlo — APPROVED** [review]
Looks good! Thanks for adding these tests Lauren!

File: tests/longstitch_tests.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+"""Tests for ntJoin"""
```
Comment (janetxinli) [inline tests/longstitch_tests.py:1]: Did you mean `LongStitch` here? :smile:

**Review by janetxinli — COMMENTED** [review]
Looks good, Lauren! Thank you for setting up these tests :)

File: tests/longstitch_tests.py (hunk 1 above)
Comment (lcoombe) [reply tests/longstitch_tests.py:1]: Oh yes good catch! :D I'm busted for my copy/pasting from the ntJoin pytests haha

Comment (janetxinli) [reply tests/longstitch_tests.py]: Hahaha we all do it!!

**Review by janetxinli — APPROVED** [review]

**Pull request merged**